packages/shared-ux/page/analytics_no_data/src/analytics_no_data_page.stories.tsx

```diff
@@ -8,13 +8,13 @@
 
 import React from 'react';
 import { action } from '@storybook/addon-actions';
+import { AnalyticsNoDataPageStorybookMocks } from '@kbn/shared-ux-page-analytics-no-data-mocks';
+import type { AnalyticsNoDataPageStorybookParams } from '@kbn/shared-ux-page-analytics-no-data-mocks';
```

Comment:
I would expect `type` import to come from `'@kbn/shared-ux-page-analytics-no-data-types'`

Reply:
You can... but I re-exported it for convenience, mostly so it's not necessary for a consumer to rely on more than one package to use a component.